include/capstone.h

```diff
@@ -27,19 +27,23 @@ extern "C" {
 #pragma warning(disable:4201)
 #pragma warning(disable:4100)
 #define CAPSTONE_API __cdecl
+#ifndef CAPSTONE_EXPORT
```

Comment:
Actually do not, for me it was enough to actually support CAPSTONE_STATIC like with MSVC, but I thought it is a far more sane idea to be able to customise that.